rust/src/transformations/make_stable_expr/expr_col/mod.rs

```diff
@@ -0,0 +1,115 @@
+use polars::prelude::*;
+
+use crate::core::{Function, MetricSpace, StabilityMap, Transformation};
+use crate::domains::ExprDomain;
+use crate::error::*;
+
+use super::DatasetOuterMetric;
+
+/// Make a Transformation that returns a `col(column_name)` expression for a Lazy Frame.
+///
+/// # Arguments
+/// * `input_domain` - Domain of the expression to be applied.
+/// * `input_metric` - The metric under which neighboring LazyFrames are compared.
+/// * `col_name` - The name of the column to be selected.
+pub fn make_expr_col<M>(
+    input_domain: ExprDomain,
+    input_metric: M,
+    expr: Expr,
+) -> Fallible<Transformation<ExprDomain, ExprDomain, M, M>>
+where
+    M: DatasetOuterMetric,
+    M::Distance: Clone + 'static,
+    (ExprDomain, M): MetricSpace,
+{
+    let Expr::Column(col_name) = expr else {
+        return fallible!(MakeTransformation, "Expected Column expression");
+    };
+    let col_name = col_name.to_string();
+
+    let mut output_domain = input_domain.clone();
+    output_domain
+        .frame_domain
+        .series_domains
+        .retain(|v| v.field.name == col_name);
+
+    // ensure that column exists in the domain
+    output_domain.active_series()?;
+
+    Transformation::new(
+        input_domain,
+        output_domain,
+        Function::new_fallible(
+            // in most other situations, we would use `Function::new_expr`, but we need to return a Fallible here
+            move |(plan, expr): &(LogicalPlan, Expr)| -> Fallible<(LogicalPlan, Expr)> {
+                if expr != &Expr::Wildcard {
+                    return fallible!(
+                        FailedFunction,
+                        "make_col has to be the first expression in the expression chain"
+                    );
+                }
+                Ok((plan.clone(), col(&*col_name)))
+            },
+        ),
+        input_metric.clone(),
+        input_metric,
+        StabilityMap::new(Clone::clone),
+    )
+}
+
+#[cfg(test)]
+mod test_make_col {
+    use crate::metrics::SymmetricDistance;
+    use crate::transformations::{polars_test::get_test_data, StableExpr};
+
+    use super::*;
+
+    #[test]
+    fn test_make_col_expr() -> Fallible<()> {
+        let (lf_domain, lf) = get_test_data()?;
+        let expr_domain = lf_domain.select();
+        let expr_exp = col("A");
+        let t_col: Transformation<_, _, _, SymmetricDistance> = expr_exp
+            .clone()
+            .make_stable(expr_domain.clone(), SymmetricDistance)?;
+        let expr_res = t_col.invoke(&(lf.logical_plan, all()))?.1;
+
+        assert_eq!(expr_res, expr_exp);
+
+        Ok(())
+    }
+
+    #[test]
+    fn test_make_col_expr_no_wildcard() -> Fallible<()> {
+        let (lf_domain, lf) = get_test_data()?;
+        let expr_domain = lf_domain.select();
+
+        let expr_exp = col("A");
+        let t_col: Transformation<_, _, _, SymmetricDistance> =
+            expr_exp.make_stable(expr_domain.clone(), SymmetricDistance)?;
```

Comment:
```suggestion
        let t_col: Transformation<_, _, _, SymmetricDistance> =
            col("A").make_stable(expr_domain.clone(), SymmetricDistance)?;
```

If `exp` is `expected`, it's a little odd because we don't end up using it in a comparison.